### PR TITLE
Refactor ballerina uninstall command

### PR DESCRIPTION
**Pull request by NatashaWso2**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/7296

  ballerina uninstall org-name/package:version

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: cli/ballerina-launcher/src/main/resources/cli-help/ballerina-uninstall.help [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+OPTIONS
+     Ballerina uninstall command takes no options.
+
+DEFAULT BEHAVIOR
+     Uninstalls a package from home repository.
+
+EXAMPLES
+     Uninstall math package
+     $ ballerina uninstall user_name/math:0.0.1
```
Comment (hemikak) [inline cli/ballerina-launcher/src/main/resources/cli-help/ballerina-uninstall.help]: Should it be org_name/math:0.0.1 ?

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/UninstallUtils.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (hemikak) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/UninstallUtils.java]: 2018

File: cli/ballerina-launcher/src/main/resources/cli-help/ballerina-uninstall.help (hunk 1 above)
Comment (NatashaWso2) [reply cli/ballerina-launcher/src/main/resources/cli-help/ballerina-uninstall.help]: Fixed with 377dcae

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/UninstallUtils.java (hunk 2 above)
Comment (NatashaWso2) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/UninstallUtils.java]: Fixed with 377dcae

**Comment by NatashaWso2** [comment]
Closing this since it was decided not to include the uninstall command

**Pull request closed without merging**